libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -157,6 +158,19 @@ bool AC_PolyFence_loader::get_item(const uint16_t seq, AC_PolyFenceItem &item)
         }
         item.radius = fence_storage.read_float(offset);
         break;
+    case AC_PolyFenceType::HOME_CIRCLE_INCLUSION: {
+        if (!AP::ahrs().home_is_set()) {
+            Debug("fence home circle missing home");
+            return false;
+        }
+        auto const home = AP::ahrs().get_home();
```

Comment:
I don't think you should be populating lat/lng here - you don't need to store these values at all.